docs/administration/troubleshooting.md

```diff
@@ -4,12 +4,36 @@
 
 ## Frequently Asked Questions (FAQs)
 
-1. I've made changes to my `ClusterLogForwarder` (CLF) instance but the collectors are not redeployed/updated. Why is that?
-    - There could be an issue with one of the inputs, outputs, or pipelines of the CLF. Check the CLF status in one of 2 ways:
+### 1. I've made changes to my `ClusterLogForwarder` (CLF) instance but the collectors are not redeployed/updated. Why is that?
+    
+   - There could be an issue with one of the inputs, outputs, or pipelines of the CLF. Check the CLF status in one of 2 ways:
         1. Streamed events of the CLF.
             
             ```$ oc describe clf --show-events=true```
 
         2. Check the `status` section of the CLF instance `YAML` output.
         
-            ```$ oc get clf -oyaml```
\ No newline at end of file
+            ```$ oc get clf -oyaml```
+
+### 2. Warning Message: `Currently ignoring file too small to fingerprint`
+If you see this warning message in your logs, here is an explanation of what it means and what actions you should take.
```

Comment:
We should also add in how the behavior of conmon impact collection and can cause the collector to generate the message.  Recall from the slack thread that conmon's behavior is to create an empty file.  This implies that if the container never produces a log message or the collector is checking the file before it has produced a log message then the warning message will appear.